app/routes/my-tickets/upcoming/list.js

```diff
@@ -56,11 +56,23 @@ export default Route.extend({
                 val  : moment().toISOString()
               }
             },
-            {
-              name : 'status',
-              op   : 'eq',
-              val  : 'placed'
-            },
+            { or: [
+              {
+                name : 'status',
+                op   : 'eq',
+                val  : 'placed'
```

Comment:
@CosmicCoder96 should we show `placed` orders in `Completed Orders` as we cannot pay for it via our portal? Or we can have a `Placed Orders` tab for it. Your views?

Reply:
@shreyanshdwivedi Let's move it to completed.